lib/go-tc/servers.go

```diff
@@ -1356,7 +1358,9 @@ type ServerV50 struct {
 	// The time at which revalidations for this server were last updated by t3c.
 	RevalApplyTime *time.Time `json:"revalApplyTime" db:"revalidate_apply_time"`
 	// The time at which revalidations were last queued for this server.
-	RevalUpdateTime   *time.Time `json:"revalUpdateTime" db:"revalidate_update_time"`
+	RevalUpdateTime *time.Time `json:"revalUpdateTime" db:"revalidate_update_time"`
+	// If the last reval apply failed for this server
+	RevalUpdateFailed *bool      `json:"revalUpdateFailed" db:"revalidate_update_failed"`
```

Comment:
These have a default value; do they need to be reference types? When would this be `nil`, and what does that mean?

Reply:
No reason